CHANGELOG.rst

```diff
@@ -6,7 +6,8 @@ This document describes changes between each past release.
 2.2.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add ``--data`` and ``--records`` options to be able to load objects
+  data without loading records. (#33)
```

Comment:
If the friday post lunch digestion let me understand this correctly, we are talking about *dump* here, and not *load* :)